### PR TITLE
Portfolio Facility

### PR DESCRIPTION
**Pull request by barisegesevgili**
## What this PR does / why we need it:

- A portfolio is: Grouping of financial assets such as stocks, bonds, commodities, currencies and cash equivalents, as well as their fund counterparts, including mutual, exchange-traded and closed funds.

- Considering the requirements, Portfolio System is implemented.

## Implemented features:
 
 - [x] Create/Delete portfolio (/create, /delete)
 
 - [x] Adding/Discarding equipment to portfolio (/addToPortfolio, /deleteFromPortfolio)
 
 - [x] Get a specific portfolio (/getPortfolio)

### CODE REVIEW

File: backend/TraderX/src/main/java/cmpe451/group6/rest/equipment/service/EquipmentService.java [hunk 1]
```diff
@@ -31,21 +30,43 @@
...
+     */
+    public double getDailyChange(String code) {
+
+        EquipmentResponseDTO equipment = this.getEquipment(code);
+
+        List<EquipmentHistoryDTO> hist = equipment.getHistoricalValues();
+
+        int histSize = hist.size();
+
+        double currentVal = equipment.getEquipment().getCurrentValue();
```
Comment (enozcan) [inline backend/TraderX/src/main/java/cmpe451/group6/rest/equipment/service/EquipmentService.java]: history might not be initialized on startup. We mush add a handler here if `hist.size == 0`

File: backend/TraderX/src/main/java/cmpe451/group6/rest/equipment/model/Equipment.java [hunk 2]
```diff
@@ -1,6 +1,7 @@
 package cmpe451.group6.rest.equipment.model;
 
 
+import cmpe451.group6.rest.portfolio.model.Portfolio;
```
Comment (enozcan) [inline backend/TraderX/src/main/java/cmpe451/group6/rest/equipment/model/Equipment.java]: Might be removed

File: backend/TraderX/src/main/java/cmpe451/group6/authorization/model/User.java [hunk 3]
```diff
@@ -1,6 +1,7 @@
 package cmpe451.group6.authorization.model;
 
 import cmpe451.group6.rest.asset.model.Asset;
+import cmpe451.group6.rest.portfolio.model.Portfolio;
```
Comment (enozcan) [inline backend/TraderX/src/main/java/cmpe451/group6/authorization/model/User.java]: Unused changes. Might be removed

**Comment by barisegesevgili** [comment]
All requests are met. Merging to backend-dev

**Pull request merged**